src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -99,31 +108,51 @@ object Arisa : ConfigSpec() {
-                        "In case of a game crash, please also attach the crash log from " +
-                        "{{[minecraft/crash-reports/crash-<DATE>-client.txt|http://hopper.minecraft.net/help/guides/finding-minecraft-data-folder/]}}."
-            )
+                "*Thank you for your report!*\n" +
+                        "However, this issue is {color:#FF5722}*Incomplete*{color}.\n" +
+                        "\n" +
+                        "Your report does not contain enough information. As such, we're unable to understand or reproduce the problem.\n" +
+                        "Please review the guidelines linked below before making further reports.\n" +
+                        "\n" +
+                        "In case of a game crash, be sure to attach the crashlog from {{[minecraft/crash-reports/crash-<DATE>-client.txt|https://minecrafthopper.net/help/guides/finding-minecraft-data-folder/]}}.\n" +
+                        "\n" +
+                        "*Quick Links*:\n" +
+                        "\uD83D\uDCD3 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- \uD83D\uDCAC [Community Support|https://discord.gg/58Sxm23] -- \uD83D\uDCE7 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- \uD83D\uDCD6 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]"
+                {color:#bbb}-- "I am a bot. This action was performed automagically! Please report any issues in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/]{color}"
+                )
         }
 
         object Crash : ModuleConfigSpec() {
             val maxAttachmentAge by optional(30)
             val crashExtensions by optional(listOf("txt", "log"))
             val duplicateMessage by optional(
-                "Duplicate of {DUPLICATE} -- " +
-                        "If you have not, please use the [search function|https://bugs.mojang.com/issues/] in the future, " +
-                        "to see if your bug has already been submitted.\r\n" +
-                        "For technical support, please use the " +
-                        "[Mojang Support Center|http://help.mojang.com/customer/portal/articles/364794-where-can-i-find-more-help-and-technical-support-]."
-            )
+                "*Thank you for your report!*\n" +
+                        "We're actually already tracking this issue in *{DUPLICATE}*, so I've resolved and linked this ticket as a duplicate.\n" +
+                        "\n" +
+                        "If you would like to add a vote and any extra information to the main ticket it would be appreciated.\n" +
+                        "\n" +
+                        "If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=MC] to see if the issue has already been mentioned.\n" +
+                        "\n" +
+                        "*Quick Links*:\n" +
```

Comment:
We always had quick links in a new line? (\n means start a new line)

Reply:
I was just going off of what neun said for that one: https://github.com/mojira/arisa-kt/pull/49#discussion_r404733878

Reply:
From helper messages:
```
*Thank you for your report!*
We're actually already tracking this issue in *LINK*, so I've resolved and linked this ticket as a duplicate.

If you would like to add a vote and any extra information to the main ticket it would be appreciated.

If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=MC] to see if the issue has already been mentioned.

*Quick Links*:
📓 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- 📖 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]
```

Reply:
Do I need to change anything or are we good?

Reply:
Looks good.
@urielsalis?

Reply:
we are good